aidial_adapter_bedrock/server/exceptions.py

```diff
@@ -85,6 +89,11 @@ def to_dial_exception(e: Exception) -> DialException:
             f"botocore.exceptions.ClientError.response: {json.dumps(response)}"
         )
 
+        if (
+            message := response.get("message")
+        ) and "One or more prompts contains filtered words" in message:
+            return content_filter_error(message)
```

Comment:
Minior suggestion:

If you want to add verbosity for user, that this is `content_filter_error`, maybe we could hide this `"One or more prompts contains filtered words" in message` also?


And make something like this:
```python
def get_content_filter_error(response: Response) -> InvalidRequestError | None:
        if (
            message := response.get("message")
        ) and "One or more prompts contains filtered words" in message:
            return InvalidRequestError(message=message, code="content_filter")
 ...

def to_dial_exception(...):
    if content_filter_error := get_content_filter_error(response):
        return content_filter_error
```

Reply:
Fixed